eventmesh-examples/src/main/java/org/apache/eventmesh/grpc/sub/CloudEventsSubscribeReply.java

```diff
@@ -0,0 +1,69 @@
+package org.apache.eventmesh.grpc.sub;
+
+import io.cloudevents.CloudEvent;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.eventmesh.client.grpc.config.EventMeshGrpcClientConfig;
+import org.apache.eventmesh.client.grpc.consumer.EventMeshGrpcConsumer;
+import org.apache.eventmesh.client.grpc.consumer.ReceiveMsgHook;
+import org.apache.eventmesh.client.tcp.common.EventMeshCommon;
+import org.apache.eventmesh.common.protocol.SubscriptionItem;
+import org.apache.eventmesh.common.protocol.SubscriptionMode;
+import org.apache.eventmesh.common.protocol.SubscriptionType;
+import org.apache.eventmesh.util.Utils;
+
+import java.util.Collections;
+import java.util.Optional;
+import java.util.Properties;
+
+@Slf4j
+public class CloudEventsSubscribeReply implements ReceiveMsgHook<CloudEvent> {
+
+    public static CloudEventsSubscribeReply handler = new CloudEventsSubscribeReply();
+
+    public static void main(String[] args) throws InterruptedException {
+        Properties properties = Utils.readPropertiesFile("application.properties");
+        final String eventMeshIp = properties.getProperty("eventmesh.ip");
+        final String eventMeshGrpcPort = properties.getProperty("eventmesh.grpc.port");
+
+        final String topic = "TEST-TOPIC-GRPC-RR";
+
+        EventMeshGrpcClientConfig eventMeshClientConfig = EventMeshGrpcClientConfig.builder()
+            .serverAddr(eventMeshIp)
+            .serverPort(Integer.parseInt(eventMeshGrpcPort))
+            .consumerGroup("EventMeshTest-consumerGroup")
+            .env("env").idc("idc")
+            .sys("1234").build();
+
+        SubscriptionItem subscriptionItem = new SubscriptionItem();
+        subscriptionItem.setTopic(topic);
+        subscriptionItem.setMode(SubscriptionMode.CLUSTERING);
+        subscriptionItem.setType(SubscriptionType.ASYNC);
```

Comment:
this should be `subscriptionItem.setType(SubscriptionType.SYNC)`, because this is request-reply mode.